### ISSUE TITLE
Bring back General Settings to the UI

### ISSUE THREAD

**Issue by te5s3rakt** [alexjustesen/speedtest-tracker#1674]
**Is your feature request related to a problem? Please describe.**
Much to my surprise, I logged into one of my Speedtest-Tracker sessions this morning and noted that it hasn't been logging for 2 months now.
Upon further reading of the repo, appears this was due to a breaking change back in June, whereby General Settings was removed from the UI, and configuration replaced with docker environment variables.

I believe this a loss and a step back to the project. Usually projects upgrade from backend configuration to frontend, not the other way around, unless essential. I don't believe (at least I couldn't determine) the reason for this change was clearly articulated.

The General Settings provided much similar configuration for lower skilled users. For example I have it running on a couple of friends and family members networks (6x) (Raspberry Pi's are great), and provide them with the ability to monitor and configure their own speed-test clients. Some of them like to check their tests, and adjust their schedules based on how they're feeling about their ISP reliability at the time. _Consequently this is how I discovered it wasn't working, a family member brought it up._

In its current state, I now have to configure everyone's clients to a fixed schedule, and change for them when requested, which isn't viable.

**Describe the solution you'd like**
I'd like to propose consideration be made to bring back General Settings to the web UI in a way that works with the modified codebase. Whatever that looks like.

**Additional context**
Doesn't need to be a high priority enhancement because, as communicated in other threads, all previous functionality still exists, just as ENV variables. But I believe this should on the roadmap for the project to achieve its maximum potential.

**Comment by EugenEule** [comment]
I also missed that feature, so I reverted to version v0.19.0, where it works perfectly. I don’t understand why the dialog was removed in v0.20.x?

**Comment by henryjb99** [comment]
I agree with both of these users. The general setting GUI option was much nicer.

**Comment by kmanwar89** [comment]
+1 on the sentiment above. Docker environment variables are great for programmability, I get it, but the convenience to set-and-forget via the UI, and make quick adjustments, was very appreciated.

**Comment by alexjustesen** [comment]
As I said in previous issues General Settings in it's previous form will not be coming back. Any thing application config based (i.e. app name, date formats etc) have been moved to environment variables so I can avoid a dependency that was causing stability and startup issues.

That being said, I'm letting the cat out of the bag here and not committing to a date but scheduling tests (yes more than 1 schedule) will come back to the UI.
